README.md

```diff
@@ -319,22 +319,20 @@ does not build `extensions` and `distribution` modules.
 
 ### Testing
 
-Take into account that the default build executes thousands of tests which may
-take a considerable amount of time. Hazelcast has 3 testing profiles:
+Take into account that the default build executes thousands of tests which may take a considerable amount of time.
```

Comment:
I don't know what are our conventions about line length for README file. I'm not sure whether we should use "longer" lines.

Reply:
Oops, reformatting on commit has kicked in, will revert it